src/Filter/IdeaStatusFilter.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace AppBundle\Filter;
+
+use ApiPlatform\Core\Bridge\Doctrine\Orm\Filter\AbstractFilter;
+use ApiPlatform\Core\Bridge\Doctrine\Orm\Util\QueryNameGeneratorInterface;
+use AppBundle\Entity\IdeasWorkshop\Idea;
+use AppBundle\Entity\IdeasWorkshop\IdeaStatusEnum;
+use AppBundle\Repository\IdeaRepository;
+use Doctrine\ORM\QueryBuilder;
+
+final class IdeaStatusFilter extends AbstractFilter
+{
+    /**
+     * @var IdeaRepository
+     */
+    private $ideaRepository;
+
+    protected function filterProperty(
+        string $property, $value,
+        QueryBuilder $queryBuilder,
+        QueryNameGeneratorInterface $queryNameGenerator,
+        string $resourceClass,
+        string $operationName = null
+    ): void {
+        if (Idea::class === $resourceClass
```

Comment:
it should be a test for stopping the execution and don't modify the query.
It's not correct,
```php
if (
  Idea::class !== $resourceClass
  || !array_key_exists($property, $this->properties) 
  || !\in_array($value, ['desc', 'asc'])
) {
  return;
}
```